integration/mediation-tests/tests-service/src/test/java/org/wso2/carbon/esb/rest/test/api/RestApiAdminServiceTestCase.java

```diff
@@ -21,34 +21,51 @@
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
+import org.wso2.carbon.rest.api.stub.RestApiAdminAPIException;
 import org.wso2.carbon.rest.api.stub.types.carbon.APIData;
 import org.wso2.carbon.rest.api.stub.types.carbon.ResourceData;
 import org.wso2.esb.integration.common.clients.rest.api.RestApiAdminClient;
 import org.wso2.esb.integration.common.utils.ESBIntegrationTest;
 
+import java.rmi.RemoteException;
+
 /**
  * Testcase for RestAPIAdminService
  */
 public class RestApiAdminServiceTestCase extends ESBIntegrationTest {
 
     private RestApiAdminClient restAdminClient;
-    private APIData api1, api2;
+    private APIData api1, api2, api3;
     private static final String tenantAPIName = "TenantServiceAPI";
     private static final String tenantDomain = "carbon.super";
+    private static final String sampleInSequence = "<inSequence>\n" + "<log level=\"custom\">\n"
+            + " <property name=\"processing sequence\" value=\"Executing sequence\"/>\n" + " </log>\n"
+            + " </inSequence>";
 
     @BeforeClass(alwaysRun = true)
     public void init() throws Exception {
         super.init();
         restAdminClient = new RestApiAdminClient(context.getContextUrls().getBackEndUrl(), getSessionCookie());
         api1 = new APIData();
         api2 = new APIData();
+        api3 = new APIData();
+
         //  sample API resource
         ResourceData resource = new ResourceData();
         String[] methods = { "POST" };
         resource.setMethods(methods);
         resource.setUrlMapping("/usage");
+
+        ResourceData resource2 = new ResourceData();
+        String[] methods2 = { "GET" };
```

Comment:
methods2 is not used?

Reply:
Thanks for noting. Fixed with a29d77f74e4ccbfae795407dcd1af1c076399bc0